src/sql_engine/src/ddl/drop_table.rs

```diff
@@ -12,33 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use crate::query::TableId;
 use kernel::SystemResult;
 use protocol::results::{QueryErrorBuilder, QueryEvent, QueryResult};
-use sqlparser::ast::ObjectName;
 use std::sync::{Arc, Mutex};
 use storage::{backend::BackendStorage, frontend::FrontendStorage, DropTableError};
 
 pub(crate) struct DropTableCommand<P: BackendStorage> {
-    name: ObjectName,
+    name: TableId,
```

Comment:
This is the drop table command. It only needs to know the table id. It should be left as TableId.

Reply:
`CreateSchemaCommand` has `SchemaCreationInfo`, I was thinking that `CreateSchemaCommand` and `CreateTableCommand` should be consistent. But now I don't know what is the difference between `SchemaId` and `SchemaCreationInfo`?

Reply:
The idea was that the Schema/.TableCreationInfo structures would store the raw schema and table names that are needed when creating the objects. Table/SchemaId would be unique identifiers for referencing these objects. Right now, these Ids are just strings but ideally, TableId would be `(u32, u32)` and SchemaId would be a `u32`.

CreateSchemaCommand and CreateTableCommand are consistent with each using their respective CreationInfo structs. Create Schema has a struct instead of a string because there are other options that can be added that we are not supporting right now.

Reply:
I missed the fact that it is drop table command